recipes/trisbm/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "nsbm" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/fvalle1/trisbm/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 268897a510cd0de903f8069376e4124544c515b1c070af6af31fea0b0ef7628c
+
+build:
+  skip: true  # [win]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

---

my package depends on graph-tool, which is not available on win.
It will not run on win.

should I use no arch anyway?

---

That's fine, the CI will fail here but once this is merged it will only ever be built on Linux. Installing will then fail with something like `graph-tool is not available for win-64` and someone can look at building `graph-tool` for Windows if they care about it.